src/views/v2/month/grid-header.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * View: Month View - Grid Header
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/month/grid-header.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+// @todo: Check if we're keeping these template tags or how is that we'll be populating data here.
+$days_of_week = tribe_events_get_days_of_week();
+global $wp_locale;
+?>
+<header class="tribe-events-calendar-month__header" role="rowgroup">
```

Comment:
Think this `role` can just be `row`?

Reply:
I've now added a new one as a wrapper for that, that its role is row